src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANGE = 100;
+    private static final String DEFAULT_COLOR = "WHITE";
+    private static int index;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private String figureType;
+    private Figure figure;
+    private String figureColor;
```

Comment:
>     private static int index;
>     private String figureType;
>     private Figure figure;
>     private String figureColor;

There is no need to have these fields at class level.

Reply:
OK!